src/copyright/agent/database.cc

```diff
@@ -288,6 +289,7 @@ std::vector<unsigned long> CopyrightDatabaseHandler::queryFileIdsForUpload(int a
 #ifdef IDENTITY_COPYRIGHT
     , agentId, agentId
```

Comment:
Please remove the extra `, agentId` parameter.

Reply:
Here https://github.com/fossology/fossology/pull/1197 they are getting replaced. anyway :)